tools/README.md

```diff
@@ -61,7 +61,7 @@ $ ./tools/upfrog.sh pango-1.40.9
 
 Copy compiled pre-built binary's SHA256 files to `release/$ARCH`.
 
-```
+```text
```

Comment:
bash instead of text?  We don't use chkfrog.sh anymore since we can no longer host on bintray for free.

Reply:
I've got a plan for that later, but as this is a pretty urgent fix, as without it PR's touching markdown will fail CI, I think it makes more sense to prioritise merging this first.